src/blocks/JobOffers.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react'
+import styled from 'styled-components'
+import PreTitle from 'components/PreTitle'
+import get from 'utils/get'
+import Grid from 'components/Grid'
+import Typography from 'components/Typography'
+import Button from 'components/Button'
+import Tag from 'components/Tag'
+import BaseCard from 'components/Card'
+
+const Section = styled.section`
+  padding: 72px 0;
+
+  @media (min-width: ${get('breakpoints.md')}) {
+    padding: 88px 0;
+  }
+
+  @media (min-width: ${get('breakpoints.lg')}) {
+    padding: 128px 0;
+  }
+
+  @media (min-width: ${get('breakpoints.xl')}) {
+    padding: 168px 0;
+  }
+`
+
+const TitleSection = styled.div`
+  grid-column: 1 / -1;
+`
+
+const Cta = styled(Button)`
+  margin-top: 20px;
+
+  @media (min-width: ${get('breakpoints.md')}) {
+    margin-top: 0;
+  }
+`
+
+const SmallSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+
+  @media (min-width: ${get('breakpoints.md')}) {
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+`
+
+const Card = styled(BaseCard)`
```

Comment:
Not sure if Card is not to generic as a naming and could be confused with a generic Card used across the whole code base. Maybe `JobOfferCard`. 

Again, not sure, feel free to ignore as the same can be said to a lot of other name choices in this PR.

Reply:
You're right, I'll rename it. Thanks 🙏